### ISSUE TITLE
mtech-report-related

### ISSUE THREAD

**Issue by cybergu0ck** [agneshraok/notes#36]
! Add barplot of time took to run the models
! Add the versions of all the packages
! Add a table of the hardware along with the cuda cores and all stuff
! make literature review on individual models.
! add stuff (unique takes) to introduction
! Make theoretical background like this:
	Convolution Operation
		stride
	Convolution Layers
	Pooling Layers
	Loss Function
	Optimiser
	Other parameters
		training steps
		valid steps
		learning rate
		epochs

	CNN architectures
! Add the formula (dice loss, entropy loss, accuracy...)	
! Add the formula for convolution image size
! try to generate activation chart and confusion matrix
! Fix the image references
